Dockerfile

```diff
@@ -1,8 +1,19 @@
-FROM alpine:3.10
+FROM nixos/nix
 
+RUN nix-channel --add https://nixos.org/channels/nixpkgs-unstable nixpkgs
+RUN nix-channel --update
+
+COPY . /usr/myapp
+WORKDIR /usr/myapp
+
+RUN nix-shell
+RUN make boots
```

Comment:
I don't think this works as nix-shell will setup the binaries only for its run and then there's nothing to do so exits and then the next `RUN` doesn't have them.

We'd need something like `RUN nix-shell --run 'make boots'